src/main/scala/mesosphere/marathon/core/launchqueue/impl/LaunchQueueActor.scala

```diff
@@ -81,8 +83,8 @@ private[impl] class LaunchQueueActor(
     import context.dispatcher
     instanceTracker.instancesBySpec().pipeTo(self)
 
-    // Using an actorMaterializer that encompasses this context will cause the stream to auto-terminate when this actor does
-    implicit val materializer = ActorMaterializer()(context)
+    // Using an Materializer that encompasses this context will cause the stream to auto-terminate when this actor does
+    implicit val materializer = Materializer(context)
```

Comment:
This means the materialized stream will outlive the actor, right. Do we want that? What do you think about materializing the stream in the pre start, as we do [here](https://github.com/mesosphere/marathon/blob/master/src/main/scala/mesosphere/marathon/core/health/impl/HealthCheckActor.scala).

Reply:
It won't outlive the actor; `Materializer(context)` ensures that the stream is terminated with the actor.

We could materialize in pre-start but I think it would have the same effect.